README.md

```diff
@@ -1,11 +1,24 @@
-# true
+# One Force API
+
+Building Application
+---
+1. Run `mvn clean install` to build your application
+2. Add db.properties file (examples below)
```

Comment:
If this is to be committed after view job roles in backend it should use local environment variables i.e. export DB_NAME = "Team1_KelvinC"